hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -1007,10 +1011,12 @@ private final class OzoneFileStatusIterator<T extends FileStatus>
      * @param p path to file/directory.
      * @throws IOException
      */
-    private OzoneFileStatusIterator(Path p) throws IOException {
+    private OzoneFileStatusIterator(Path p, Function<FileStatus, T> transformFunc, boolean lite) throws IOException {
```

Comment:
We may update the Javadoc to incorporate the newly added parameters.

Reply:
done